src/web/PresentationControls.tsx

```diff
@@ -46,14 +48,15 @@ export function PresentationControls({
   const [spring, api] = useSpring(() => ({ scale: 1, rotation: rInitial, config }))
   React.useEffect(() => void api.start({ scale: 1, rotation: rInitial, config }), [rInitial])
   React.useEffect(() => {
-    if (global && cursor) gl.domElement.style.cursor = 'grab'
-  }, [global, cursor, gl.domElement])
+    if (global && cursor && enabled) gl.domElement.style.cursor = 'grab'
```

Comment:
doesn't this need a cleanup in this case?

Reply:
that is true!